linter.py

```diff
@@ -73,3 +82,52 @@ def ensure_plugin_installed(self) -> bool:
 		# the environment. Silently, do not notify and disturb the user!
 		self.notify_unassign()
 		raise PermanentError()
+
+def guess_cwd(view):
+	if view.file_name():
+		return os.path.dirname(view.file_name())
+	elif len(view.window().folders()):
+		return view.window().folders()[0]
+
+def run_cmd(cmd, data, view):
+	cwd = guess_cwd(view)
+	if isinstance(data, str):
+		data = data.encode()
+
+	proc = subprocess.Popen(
```

Comment:
Should use `subprocess.run()`.

Reply:
Yes but Sublime Text is using Python v2, and `subprocess.run()` is available in Python v3.

Reply:
For example, in `gofmt`:
https://github.com/noonat/sublime-gofmt/blob/master/gofmt.py#L93

Reply:
We can drop Python 2 support. I'm not interested in supporting older Sublime versions anyway.

Reply:
I have the latest version on Xubuntu 20.04:
![image](https://user-images.githubusercontent.com/12686176/82732625-ac92cd80-9ce4-11ea-9913-a4a1571e7d77.png)
The code doesn't work with `subprocess.run()`, so I checked the Python version and it said 2.7, I went to the documentation and `subprocess.run()` is available from 3.5.

Not sure what should I do but I have everything very default, so a lot of people has Sublime with Python 2, right?

When I researched about `cwd` I noticed that they use `subprocess.Popen()` as well.

Reply:
Actually, Sublime 3 ships with Python 3.3: http://www.sublimetext.com/docs/3/porting_guide.html

So yes, it does indeed look like it's too old...

Can you add a TODO comment about using `.run()` when Sublime eventually upgrades to 3.5?

Reply:
When Sublime get updated I can do the change, I will try to remember it or you can let me know.

Reply:
I don't think it will happen anytime soon. I remember reading on the forums that there was a reason it was outdated.